src/app/src/App.js

```diff
@@ -1,25 +1,18 @@
-import React from 'react';
-import logo from './logo.svg';
+import { BrowserRouter, Navigate, Routes, Route } from 'react-router-dom';
+
 import './App.css';
+import Welcome from './pages/Welcome';
+import Main from './pages/Main';
 
 function App() {
     return (
-        <div className='App'>
-            <header className='App-header'>
-                <img src={logo} className='App-logo' alt='logo' />
-                <p>
-                    Edit <code>src/App.js</code> and save to reload.
-                </p>
-                <a
-                    className='App-link'
-                    href='https://reactjs.org'
-                    target='_blank'
-                    rel='noopener noreferrer'
-                >
-                    Learn React
-                </a>
-            </header>
-        </div>
+        <BrowserRouter>
+            <Routes>
+                <Route path='/welcome' element={<Welcome />} />
+                <Route path='/main' element={<Main />} />
```

Comment:
Minor thing, but consider renaming this to `draw` or something like to indicate the primary purpose of the page.

Reply:
Fixed in [b573a01](https://github.com/azavea/iow-boundary-tool/pull/21/commits/b573a0112f709d70d86a30e7edf6403bac962574)